Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -11,7 +11,7 @@ def initialize
         end
 
         sig { params(path: T.nilable(Pathname)).returns(Integer) }
-        def which(path)
+        def index_of(path)
```

Comment:
This method became an incompatible override once `Kernel.which` was typed, so it has been renamed (without a ton of thought given to the name, I'm open to alternatives).